### ISSUE TITLE
Converting SVG resulted in empty `.tsx` file

### ISSUE THREAD

**Issue by jonesaustindev** [teamairship/airfoil#35]
## Observed results

Running convert on two svgs resulted in one working and the other creating a blank `.tsx` file. The only differences I see between the two svg files I used, is the `RepeatIcon.svg` below has transform, while the other does not. Possibly that is the issue?

### Logs

```
// RepeatIcon.svg
<svg xmlns="http://www.w3.org/2000/svg" width="27.413" height="15.42" viewBox="0 0 27.413 15.42"><defs><style>.a{fill:#a0a0a0;}</style></defs><g transform="translate(0 -4)"><path class="a" d="M19.707,14.28V6.57A2.573,2.573,0,0,0,17.137,4H6L9.427,7.427H16.28V14.28H12.853l5.14,5.14,5.14-5.14Z" transform="translate(4.28 0)"/><path class="a" d="M6.853,15.993V9.14H10.28L5.14,4,0,9.14H3.427v7.71A2.573,2.573,0,0,0,6,19.42H17.133l-3.427-3.427Z" transform="translate(0 0)"/></g></svg>
```

## Steps to reproduce

1. Put `RepeatIcon.svg` in `app/assets/svg`
2. Run `airfoil convert svg`

**Comment by townofdon** [comment]
Was able to reproduce - this is very strange. Works on 
https://react-svgr.com/playground

**Comment by townofdon** [comment]
Issue ended up being the inline `<style>` tag. Fix imminent in #37 .

**Comment by townofdon** [comment]
Should be fixed in version `0.3.1` - https://github.com/teamairship/airfoil/releases/tag/v0.3.1
